controls/radmap/features/providers/bing-rest-map-provider/search-location.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Search Location
+page_title: Search Location
+description: Search Location
+slug: radmap-features-providers-bing-rest-map-provider-search-location
+tags: bing,map,provider,rest,location api, search
+published: True
+position: 2
+---
+
+# Search Location
+
+The RadMap control allows the user to implement search functionality, that will let the user search for a specific place on the map.  The search is performed through an __IBingRestSearchLocationProvider__ interface, which communicates with the respective map provider's services. The __BingRestMapProvider__ implements this interface. In the following sections of this article we will demonstrate how you can implement this functionality.
+
+>The RadMap search functionality wraps the [Location API](https://msdn.microsoft.com/en-us/library/ff701715.aspx) of the Bing Rest Map services.
+
+## Find a place by given Location or Query
+
+In the following examples, we will demonstrate how to find a specific location by given address or location. To do it you can create an instance of the __BingRestSearchLocationRequest__ object which the __SearchLocationAsync()__ method of the __BingRestMapProvider__ uses as a parameter and set its __Query__ property. Then you can subscribe to the __SearchLocationCompleted__ and __SearchLocationError__ events of the. The __SearchLocationError__ event is the place for handling any kind of errors that occur during the search operation.  You can take a look at the following steps:
+
+* Set the __ApplicationId__ property of the __BingRestMapProvider__. This property represents a key, which allows you to use the Bing Rest Maps services.
+	
+>Without supplying a valid key you won't be able to visualize the map inside the RadMap control. In order to learn how to obtain one, please read [Accessing the Control Using a Bing Maps](http://msdn.microsoft.com/en-us/library/ee681900.aspx).
```

Comment:
Those note will be rendered outside of the list item. Move it with a single tab so it is displayed in the list item.